README.md

```diff
@@ -122,6 +123,72 @@ From the application directory run:
 bin/console fom:user:resetroot
 ```
 
+## Docker
+
+Run this command to start a local Mapbender instance in a docker container using an internal SQLite database.
+
+```bash
+docker run -p 80:8080 mapbender/mapbender
+```
+
+If you want Mapbender to stay up and running in background add the `-d` option.
+
+### Port 8080
+
+The container is accepting HTTP requests on port `8080` by default.
+
+### Env variables
+
+You can use environment variables to adjust the configuration of the Mapbender docker container.
+
+- `APP_ENV` Symfony app environment. (`prod`, `dev`, `test`)
+- `MAILER_DSN` Smtp server configuration. (`smtp://user:pass@smtp.example.com:25`)
+- `MAPBENDER_DATABASE_URL` configure Mapbender to connect to another database as shown in the upcoming example. (find examples at [https://symfony.com/doc/current/doctrine.html#configuring-the-database](https://symfony.com/doc/current/doctrine.html#configuring-the-database))
+
+### external database 
+
+To run a Mapbender docker container configured to use an external database use the `MAPBENDER_DATABASE_URL` environment variable.
+
+1.  Create the docker network
+
+    `docker network create -d bridge mapbender`
+
+2.  Start the PostGIS database
+
+    `docker run --name db --network mapbender -p 55432:5432 -e POSTGRES_USER=pguser -e POSTGRES_PASSWORD=pgpass -e POSTGRES_DB=postgres postgis/postgis:14-3.4`
+
+3.  Start Mapbender
+
+    `docker run --name mapbender --network mapbender -p 80:8080 -e MAPBENDER_DATABASE_URL="pgsql://pguser:pgpass@db:5432/postgres" mapbender/mapbender`
+
+4.  Create Mapbender database schema
+
+    `docker exec mapbender php application/bin/console doctrine:schema:create`
```

Comment:
add `--force` to the command, otherwise it will only print the required sql statements to the console